### ISSUE TITLE
Make sure we can distinguish activity of an UGent profile by object class

### ISSUE THREAD

**Issue by mietcls** [ugent-library/old-people-service#15]
# Enhancement

Make sure we can distinguish activity of an UGent profile by object class (and expiration date? `TBD`).

This info needs to flow through to the Getperson and SearchPeople.

## Context

Right now, we can show if someone is active in LDAP or not in Biblio. We do this by adding a label `Active UGent member`. But we do distinguish what type of activity this is (e.g. Student, researcher, alumnus, ...). This can cause confusion and lead to incorrect assignments when affiliating people to a record.

<img width="1844" alt="Scherm­afbeelding 2023-05-05 om 17 18 16" src="https://user-images.githubusercontent.com/1572885/236499204-1c5c966c-1ea3-44be-88e4-9074d0159f98.png">

To be refined:
- Which statuses are there?
- Which ones are important to us?
- What does the expiration date mean in which context?

Related case: https://github.com/ugent-library/biblio-backoffice/issues/1087

To take into account:
> The `Active` property is woven throughout the backoffice database. It will take some time to flesh that out.

## Implementation suggestion

To be refined

## Next steps

- [ ] Make visualisation in back-office

**Comment by mietcls** [comment]
@nicolasfranck do you have any input here?

**Comment by nicolasfranck** [comment]
@mietcls yes we do collect this data in [object_class](https://github.com/ugent-library/people-service/blob/main/models/person.go#L28). The data is only available in the ugent ldap, more
specifically attribute `objectclass`.

This data is retrieved

* during import of student records (as they originate from ldap directly)
* after import of staff from gismo by doing an extra lookup in ldap
